### PR TITLE
Define AS3 info service

### PR DESCRIPTION
**Pull request by zhaoqin-github**
This code change only add AS3 info service. Other services will be added later. Since LB4 currently does not support to stub service object. We will defer the unit test of AS3 service code.

### CODE REVIEW

File: app/waf/src/controllers/ping.controller.ts [hunk 1]
```diff
@@ -1,5 +1,9 @@
 import {Request, RestBindings, get, ResponseObject} from '@loopback/rest';
 import {inject} from '@loopback/context';
+import {AS3Service} from '../services';
+
+const AS3_HOST: string = process.env.AS3_HOST || 'localhost';
+const AS3_PORT: number = Number(process.env.AS3_PORT) || 8443;
```
Comment (zongzw) [inline app/waf/src/controllers/ping.controller.ts]: We may have more configuration items in future, just thinking if we could separate them to a json-file datasource/repository/model/(controller if necessary for configurable from api) for all configuration?

File: app/waf/src/datasources/as3.datasource.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  "options": {
+    "headers": {
+      "accept": "application/json",
+      "content-type": "application/json"
+    },
+    "strictSSL": false
+  },
+  "operations": [
+    {
+      "template": {
```
Comment (zongzw) [inline app/waf/src/datasources/as3.datasource.json]: can we add a timeout property? 
ping for as3 information should be over very quickly. waiting for default 2min timeout may be too long..

Comment (zhaoqin-github) [reply app/waf/src/datasources/as3.datasource.json]: I do not find a way to specify timeout for outbound calls. Do you know how to specify it?

Comment (zongzw) [reply app/waf/src/datasources/as3.datasource.json]: Don't know yet.

**Pull request merged**